### PR TITLE
web: Remove APIv2

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

**Comment by brian-brazil** [comment]
Tests are failing.

File: vendor/modules.txt [hunk 1]
```diff
@@ -576,11 +570,8 @@ google.golang.org/appengine/urlfetch
 # google.golang.org/genproto v0.0.0-20200815001618-f69a88009b70
 ## explicit
 google.golang.org/genproto/googleapis/api/annotations
-google.golang.org/genproto/googleapis/api/httpbody
 google.golang.org/genproto/googleapis/rpc/status
-google.golang.org/genproto/protobuf/field_mask
 # google.golang.org/grpc v1.29.1
-## explicit
 google.golang.org/grpc
```
Comment (brian-brazil) [inline vendor/modules.txt:575]: Is something else pulling this in now?

**Comment by brian-brazil** [comment]
https://prometheus.io/docs/prometheus/latest/querying/api/#tsdb-admin-apis also needs updating.

File: vendor/modules.txt (hunk 1 above)
Comment (roidelapluie) [reply vendor/modules.txt:575]: Yes I figured out that it is now pulled by the protobuf stack.

**Comment by roidelapluie** [comment]
It seems like it will not solved cortex issues unless we block upgrading all the protobuf and google golang api's.

**Comment by brian-brazil** [comment]
It'll still resolve the cmux issue, and I imagine that Cortex isn't depending on GCE.

**Comment by roidelapluie** [comment]
tests are passing

**Comment by brian-brazil** [comment]
:+1:

**Pull request merged**